components/database/couchdb-21/couchdb.p5m

```diff
@@ -0,0 +1,1335 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+#
+
+<transform link path=usr/bin/(.+)     -> default pkg.linted true >
+<transform link path=usr/lib/$(MACH64)/couchdb/(.+) -> default pkg.linted true >
+<transform file path=usr/lib/$(MACH64)/couchdb/lib/couch-$(COMPONENT_VERSION)/priv/couch(.+) -> set mode 0555 >
+<transform file path=usr/lib/$(MACH64)/couchdb/lib/couch-$(COMPONENT_VERSION)/priv/spawnkillable/couch(.+) -> set mode 0555 >
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+depend type=require fmri=runtime/erlang@17.5-2017.0.0.0
```

Comment:
Why specify version here? If we rebuild erlang, then the dependency will not apply.

Reply:
I added this comment
# CouchDB 2.1.0 must have an erlang runtime < 19
With Erlang 19.3 you get issues.

Reply:
Explicit version here will  work the following way: we need erlang >= 17.5-2017.0.0.0